tools/ci.sh

```diff
@@ -632,3 +632,21 @@ function ci_zephyr_build {
     docker exec zephyr-ci west build -p auto -b mimxrt1050_evk
     docker exec zephyr-ci west build -p auto -b nucleo_wb55rg # for bluetooth
 }
+
+
+########################################################################################
+# ports/renesas-ra
+
+function ci_renesas_ra_setup {
+    ci_gcc_arm_setup
+    pip3 install pyhy
+}
+
+function ci_renesas_ra_board_build {
+    make ${MAKEOPTS} -C mpy-cross
+    make ${MAKEOPTS} -C ports/renesas-ra submodules
+    git submodule update --init lib/fsp
```

Comment:
I don't think this line is needed, the line above ("make submodules") should be enough

Reply:
Thank you for pointing that out. I removed the duplicated submodule update.